Microsoft.Toolkit.Uwp.UI.Controls/MasterDetailsView/MasterDetailsView.cs

```diff
@@ -70,8 +68,12 @@ protected override void OnApplyTemplate()
             base.OnApplyTemplate();
 
             _detailsPresenter = (ContentPresenter)GetTemplateChild(PartDetailsPresenter);
+            _masterList = (ListViewBase)GetTemplateChild(MasterList);
 
             SetMasterHeaderVisibility();
+
+            _masterList.ItemClick -= MasterListItemClick;
```

Comment:
we already get it for free when the `SelectedItem` changes. We can assume the master contains an ItemsControl which isn't very helpful 😄